acceptance/lib/pxp-agent/task_helper.rb

```diff
@@ -31,14 +31,14 @@ def run_task(broker, targets, task, filename, input = {}, max_retries = 30, quer
   end
 end
 
-def create_task_on(agent, mod, task, body)
```

Comment:
Since we're passing body to the task, why not calculate SHA as part of create_task_on?